civitai-downloader-v2/tests/test_phase6_complete.py

```
@@ -0,0 +1,672 @@
+            monitor = UptimeMonitor(data_dir=Path(temp_dir))
+            yield monitor
+    
+    @pytest.mark.asyncio
+    async def test_circuit_breaker_functionality(self, circuit_breaker):
+        """Test circuit breaker open/close functionality."""
+        # Mock function that fails
+        async def failing_function():
+            raise Exception("Test failure")
+        
+        # Test normal failure counting
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)